src/pages/home/Home.jsx

```diff
@@ -8,64 +8,117 @@ import FitnessCenterTwoToneIcon from '@mui/icons-material/FitnessCenterTwoTone';
+              <div>Mysli na sebe i na mateřské dovolené.</div>
+              <div> Buď v kondici. </div>
+              <div> Buď svá. </div>
+            </Typography>
+          </Stack>
+          <Box
+            sx={{
+              width: 450,
+              color: 'white',
+              p: 2,
+              position: 'relative',
+              top: 155,
+              zIndex: 2,
+              backgroundColor: 'rgba(193,255,200,0.2)',
+            }}
+          >
+            <p className="motivace">
+              Cvič tam, kde ti to vyhovuje a co tvé tělo potřebuje. Stravuj se
+              tak, aby Ti tvé tělo bylo vděčné a dosáhla jsi svých cílů. Vyplň
+              krátký dotazník a zjisti, jak na to.
+            </p>
+          </Box>
 
-        <Stack spacing={2} alignItems="center" my={8}>
-          <Button variant="contained" href={Urls.Questions}>
-            Pojď na to
-          </Button>
-        </Stack>
+          <Stack
+            spacing={2}
+            alignItems="center"
+            my={8}
+            position="absolute"
+            bottom={5}
+            width="100%"
+            left={0}
+            right={0}
+            zIndex={2}
+          >
+            <Button variant="contained" href={Urls.Questions} size="large">
+              Pojď na to
+            </Button>
+          </Stack>
+        </Container>
       </Box>
-      <Typography variant="h4" m={3}>
-        Zjisti:
-      </Typography>
-      <Stack direction="row" justifyContent="space-around">
-        <Stack alignItems="center">
-          <FitnessCenterTwoToneIcon fontSize="large" color="primary" />
-          <Typography variant="h6">Jak si tréninky nakombinovat.</Typography>
-        </Stack>
-        <Stack alignItems="center">
-          <EggAltTwoToneIcon fontSize="large" color="primary" />
-          <Typography variant="h6">Jak se stravovat</Typography>
-        </Stack>
-        <Stack alignItems="center">
-          <MoodTwoToneIcon fontSize="large" color="primary" />
-          <Typography variant="h6">Jak být v pohodě a šťastná</Typography>
+
+      <Container>
+        <Typography variant="h4" m={3}>
+          Zjisti:
+        </Typography>
+        <Stack direction="row" justifyContent="space-around">
+          <Stack alignItems="center">
+            <FitnessCenterTwoToneIcon fontSize="large" color="primary" />
+            <Typography variant="h6">Jak si tréninky nakombinovat.</Typography>
+          </Stack>
+          <Stack alignItems="center">
+            <EggAltTwoToneIcon fontSize="large" color="primary" />
+            <Typography variant="h6">Jak se stravovat</Typography>
+          </Stack>
+          <Stack alignItems="center">
+            <MoodTwoToneIcon fontSize="large" color="primary" />
+            <Typography variant="h6">Jak být v pohodě a šťastná</Typography>
+          </Stack>
         </Stack>
-      </Stack>
-    </Container>
+      </Container>
```

Comment:
Add another call-to-action `Button` under the benefits that links to Questions, but with different text than in banner.